backend/apps/tickets/validation.py

```
@@ -0,0 +1,8 @@
+ALLOWED_TYPES = ['image/jpeg', 'image/png', 'application/pdf']
+MAX_FILE_SIZE = 5 * 1024 * 1024  # 5 MB
+
+def validate_file(file):
+    if file.content_type not in ALLOWED_TYPES:
+        raise ValueError("Unsupported file type.")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)